vroom/clangformat.vroom

```diff
@@ -186,8 +186,10 @@ like line2byte() and :goto being buggy in that case.
   % int f()     {<CR>
   |    int i=1;<CR>
   |   return 1234567890; }<CR>
-  :call prop_type_add('keyword', {})
-  :call prop_add(1, 1, {'length': 3, 'type': 'keyword'})
+  :if has('textprop')
```

Comment:
Nit: Add a note to clarify we're not verifying anything interesting in the `!has('textprop')` case. Something like this:
```suggestion
This skips the textprop case if not supported (meaning this block won't do any
interesting verification when run in older versions of vim).
  :if has('textprop')
```